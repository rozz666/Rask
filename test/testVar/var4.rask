main() -> void
{
    var y = x;
    print(y);
}
