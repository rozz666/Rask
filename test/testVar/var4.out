var4.rask:3:13: error: unknown identifier 'x'

